packages/terra-clinical-result/src/common/other/_KnownNoData.jsx

```diff
@@ -35,9 +36,11 @@ const KnownNoData = (props) => {
     <span
       {...customProps}
       className={templateClassnames}
-      aria-label={intl.formatMessage({ id: 'Terra.clinicalResult.resultNoData' })}
     >
-      --
+      <span aria-hidden>
```

Comment:
Just curious, is there any difference between `<span aria-hidden>` and `<span aria-hidden="true">` from the accessibility tooling perspective? I notice the aria-hidden is usually set with a value in other places.

Reply:
I was given an error message from the linter when I tried explicitly setting it to true that said "Value must be omitted for boolean attributes". So basically it looks like it just defaults to true so we don't have to set it.

Reply:
Hey so I looked into this more this morning, I made a change to set it to true explicitly. aria-hidden isn't a boolean but a state that can have three different values. It defaults to 'undefined' which is working in my case for the span but is not reliable - anyways here's some more info: https://stackoverflow.com/questions/51255045/is-presence-of-aria-hidden-sufficient-or-is-value-set-to-true-required-aria-h